pkg/aws/ec2.go

```diff
@@ -0,0 +1,27 @@
+package aws
+
+import (
+	"context"
+	"testing"
+
+	awstypes "github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/gruntwork-io/terratest/modules/aws"	
+)
+
+func GetNatsByVpcIdE(t *testing.T, vpcId string, awsRegion string) ([]types.NatGateway, error) {
```

Comment:
consider renaming to `GetNatGatewaysByVpcIdE` as `nats` could be confusing.

Reply:
Done